Product.cs

```diff
@@ -4,14 +4,14 @@
 using System.Text;
 using System.Threading.Tasks;
 
+
 namespace Simple_Inventory_Management_System
 {
     class Product
     {
         private string name;
```

Comment:
we can benefit from this style to reduce the code changes. 
```suggestion
        public string name {get; set;}
```

same thing for the below fields.